### PR TITLE
Feature/geolocation

### PR DESCRIPTION
**Pull request by mdflynn**
### Description

- Adds city/state conversion to lat/long
- Adds image upload widget

### Closes issue(s)

#7 #29 

### Testing
n/a
### Screenshots
n/a
### Changes include
- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

### Checklist
- [ ] I have written tests for this code (happy & sad)
- [ ] I have updated the Readme
- [ ] I ran full test suite - all tests passing 
- [x] Lint code

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +25,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>UFOMfG</title>
```
Comment (jdxsmith) [inline public/index.html:28]: Typo?  Or is the extra 'f' on purpose?

File: src/Components/ReportForm/ReportForm.js [hunk 2]
```diff
@@ -103,6 +135,7 @@ const ReportForm = () => {
             <option value="abduction">Abduction</option>
           </select>
         </label>
+        <UploadWidget data={handleImageUpload} />
```
Comment (jdxsmith) [inline src/Components/ReportForm/ReportForm.js:138]: Nice use of the upload widget, this seems like the best way to upload an image

File: src/api.js [hunk 3]
```diff
@@ -38,4 +38,19 @@ export const postSighting = (sighting, dispatch) => {
...
+export const geolocateUser = (city, state) => {
+  const formatCity = city.split(" ").join("+");
+  const formatState = state.split(" ").join("+");
+  return fetch(
+    `https://maps.googleapis.com/maps/api/geocode/json?address=${formatCity},+${formatState}&key=${process.env.REACT_APP_API_KEY}`
+  ).then((response) => response.json());
+};
+
+// to display city state data on comment section with lat/long only storage
+export const reverseGeolocateUser = (lat, long) => {
```
Comment (jdxsmith) [inline src/api.js:50]: Good addition, this will be helpful

**Review by jdxsmith — APPROVED** [review]
Nice work on the image upload and the geolocation.  Looks great!

**Review by peytonjo — APPROVED** [review]
Looks great, I would say those changes made that Jordon commented on it'll be good. Especially making sure we change that UFOMfG typo

**Pull request merged**